src/i18n/langResources/lang.de.ts

```diff
@@ -27,6 +27,7 @@ export const langDe: ILangResource = {
   tsOfficialPresetMessage: 'Offizielles TS Icon wird',
   jsonOfficialPresetMessage: 'Offizielles JSON Icon wird',
   hideFoldersPresetMessage: 'Ordner Icons werden',
+  foldersAllDefaultIconPresetMessage: 'Ordner Icons Auflegemaske werden',
```

Comment:
Or maybe we change the implementation of the message construction so it can compensate for such scenarios. That we leave though to slightly more message keys that change only on a word.
@robertohuertasm You opinion?